docs/wiki/quick_start.md

```diff
@@ -106,6 +106,27 @@ There are two options to be create a wallet, first is using the jsng shell where
 
 Now that the wallet is setup, you are ready for your first deployment.
 
+## Adding tokens to a stellar wallet
+[Press here](https://github.com/threefoldfoundation/info_gridmanual/blob/development_capacitypool/src/2_gettingstarted/getting_started_get_tft.md)
+
+## Create your Capacity pool
+Before we get to deploy the actual Ubuntu container, we first need to create your capacity pool for any solutions use it.
```

Comment:
how about : `Before we get to deploying the actual Ubuntu container, we need to create a capacity pool which will be used by any solutions deployed.`
maybe explain a bit what is a capacity pool?